inc/compat/woocommerce.php

```diff
@@ -183,6 +183,14 @@ function primer_wc_shop_title( $title ) {
 
 	}
 
+	if ( is_product() ) {
+
+		$object = get_post_type_object( 'product' );
+
+		$title = $object->labels->singular_name;
```

Comment:
@EvanHerman We should be using `get_post_type_labels()` so that Primer will honor the `post_type_labels_product` filter, which a developer could very well be using.

cf. https://developer.wordpress.org/reference/hooks/post_type_labels_post_type/

```php
$labels = get_post_type_labels( get_post_type_object( 'product' ) );
$title  = ! empty( $labels->singular_label ) ? $labels->singular_label : $title;
```